lib/util/esWrapper.js

```diff
@@ -114,7 +114,8 @@ const errorMessagesMapping = [
 class ESWrapper {
 
   constructor(client) {
-    this.client = client;
+    t
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Add a semicolon at the end of this statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ASemicolon)